reference/generators/cmake.rst

```diff
@@ -118,16 +118,24 @@ When other library requires ``MyLib`` and uses the cmake generator:
 Methods available in conanbuildinfo.cmake
 -----------------------------------------
 
-+--------------------------------+-------------------------------------------------------------------------------------------+
-| NAME                           | DESCRIPTION                                                                               |
-+================================+===========================================================================================+
-| conan_basic_setup()            |  Setup all the CMake vars according to our settings with the global approach (no targets) |
-+--------------------------------+-------------------------------------------------------------------------------------------+
-| conan_basic_setup(TARGETS)     |  Setup all the CMake vars by target (only CMake > 3.1.2)                                  |
-+--------------------------------+-------------------------------------------------------------------------------------------+
-| conan_target_link_libraries()  |  Helper to link all libraries to a specified target                                       |
-+--------------------------------+-------------------------------------------------------------------------------------------+
-
++-----------------------------------+-------------------------------------------------------------------------------------------+
+| NAME                              | DESCRIPTION                                                                               |
++===================================+===========================================================================================+
+| conan_basic_setup()               |  Setup all the CMake vars according to our settings with the global approach (no targets) |
```

Comment:
I would say this is confusing, as it seems several calls to ``conan_basic_setup()`` can be made, to setup different things. I will use just one row for one method, then describe the different arguments in the description.